modules/add_person.rb

```diff
@@ -0,0 +1,21 @@
+def char_to_bool(char)
+  if char.downcase == 'y'
+    true
+  elsif char.downcase == 'n'
+    false
+  end
+end
+
+def add_student(age, name)
+  print 'Has parent permission? [Y/N]: '
+  $stdout.flush
+  par_perm = char_to_bool(gets.chomp)
+  Student.new(age, name, par_perm)
+end
```

Comment:
There's an error in creating a student and a teacher. This is an `ArguementError` therefore there is something wrong with the way you pass in the arguements when calling the `Student` and `Teacher` classes. Kindly look into the way you handle the Student and Teacher parameters.

Since I cannot create any Person, kindly make sure that when the issues are resolved the other basic UI functionalities related to a Person like List all people, Create a rental, and List all rentals for  given person id work as expected.

Reply:
Hello @KDlamini, thank you for the feedback. I figured this is caused by the class default parameters such as parent_permision Rubocop making me use : true instead of = which causes and issue when running the program. Do you have any recommendations how can I fix this?